packages/rocketchat-lib/server/models/Users.js

```diff
@@ -614,6 +614,18 @@ class ModelUsers extends RocketChat.models._Base {
 		return this.update({ _id }, update);
 	}
 
+	updateDefaultStatus(_id, statusDefault) {
+		return this.update({
+			_id,
+			statusDefault: { $ne: statusDefault },
+		}, {
+			$set: {
+				statusDefault,
+				_updateAt: new Date(),
```

Comment:
The _updateAt will be set by _BaseDb already here https://github.com/RocketChat/Rocket.Chat/blob/3564f831e03b70a342b58e2b06143ce6d2b2f777/packages/rocketchat-lib/server/models/_BaseDb.js#L76

Reply:
it was not updating it on my tests :expressionless: I'll give it one more try

Reply:
you were right.. it was a misspelling :man_facepalming: